lib/helpers.js

```diff
@@ -16,7 +16,7 @@ function sriDigest(file, fromString) {
 }
 
 function selectedTheme(config, selected) {
-    if (typeof selected === 'undefined' || selected === 'undefined') {
```

Comment:
I'm pretty sure there was a case where I was returning `'undefined'` as a string for selected. Might be long gone now though.